src/nix/formatter-run.md

```diff
@@ -8,6 +8,9 @@ Flags can be forwarded to the formatter by using `--` followed by the flags.
 
 Any arguments will be forwarded to the formatter. Typically these are the files to format.
 
+The environment variable `FLAKE_DIR` will be set to the absolute path to the
```

Comment:
This should be something like `NIX_FLAKE_DIR` since we want all out environment variables to be prefixed with `NIX_`.

Reply:
I've updated this to `NIX_FLAKE_DIR`